### PR TITLE
Improve sign in/register screens appearance on mobile 

### PR DESCRIPTION
**Pull request by janfaracik**
The updated sign in/register screens currently has a few issues on mobile, this PR fixes those:

* Unnecessary padding on the left/right sides making it cramped
* Textboxes text is a touch too small, causing iOS to zoom in when focused
* Missing the niceties that the main `styles.css` file has, like a custom selection colour and `-webkit-tap-highlight-color: transparent;`

**Before**

**After**

### Testing done

* The screens work the same as before

### Proposed changelog entries

- Improve sign in/register screens appearance on mobile 

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7995"></a>

### CODE REVIEW

File: war/src/main/scss/base/core.scss
Comment (janfaracik) [inline war/src/main/scss/base/core.scss:1]: Moved these selectors into their own file so they can be used on simple-pages too.

File: war/src/main/scss/base/core.scss
Comment (NotMyFault) [reply war/src/main/scss/base/core.scss:12]: Can't hurt 🤷

Comment (timja) [reply war/src/main/scss/base/core.scss:12]: @janfaracik reminder about this

Comment (janfaracik) [reply war/src/main/scss/base/core.scss:12]: https://github.com/jenkinsci/jenkins/pull/8084

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
